lib/libmbedtls/mbedtls/include/mbedtls/bignum.h

```diff
@@ -196,6 +203,8 @@ extern void *mbedtls_mpi_mempool;
  */
 void mbedtls_mpi_init( mbedtls_mpi *X );
 void mbedtls_mpi_init_mempool( mbedtls_mpi *X );
+void mbedtls_mpi_init_static( mbedtls_mpi *X , mbedtls_mpi_uint *p,
+                              int sign, size_t alloc_size, size_t nblimbs);
```

Comment:
This needs a specific comment, especially since (as I understand it) the sentence "This just makes it ready to be set or freed, but does not define a value for the MPI" is not true here.